services/cache-headers.js

```diff
@@ -0,0 +1,92 @@
+'use strict'
+
+const assert = require('assert')
+const coalesce = require('../lib/coalesce')
+
+const serverStartTimeGMTString = new Date().toGMTString()
+const serverStartTimestamp = Date.now()
+
+const numRegex = /^[0-9]+$/
+function isInt(number) {
+  return number !== undefined && numRegex.test(number)
```

Comment:
Why can't we just use `Number.isInteger` here?

Reply:
Oh right. We're looking for a string that looks like an int, not an int. Also `Number.isInteger` will accept negative numbers, whereas this will only accept >=0. Perhaps this function is actually testing the right thing but `isInt()` isn't the right name for it.